llvm/utils/TableGen/CallingConvEmitter.cpp

```diff
@@ -125,6 +126,56 @@ void CallingConvEmitter::emitCallingConv(const Record *CC, raw_ostream &O) {
 
 void CallingConvEmitter::emitAction(const Record *Action, indent Indent,
                                     raw_ostream &O) {
+
+  auto EmitRegList = [&](const ListInit *RL, const StringRef RLName) {
+    O << Indent << "static const MCPhysReg " << RLName << "[] = {\n";
+    O << Indent << "  ";
+    ListSeparator LS;
+    for (const Init *V : RL->getValues())
+      O << LS << getQualifiedName(dyn_cast<DefInit>(V)->getDef());
```

Comment:
`dyn_cast` -> `cast`. If the `dyn_cast` fails it's a null pointer dereference so I'm assuming it shouldn't fail?

Reply:
Correct, it should not fail. Done.